diff --git a/node_modules/@mui/system/createStyled.js b/node_modules/@mui/system/createStyled.js
index 46b0474..2e754cc 100644
--- a/node_modules/@mui/system/createStyled.js
+++ b/node_modules/@mui/system/createStyled.js
@@ -114,10 +114,8 @@ function createStyled(input = {}) {
     const skipSx = inputSkipSx || false;
     let label;
 
-    if (process.env.NODE_ENV !== 'production') {
-      if (componentName) {
-        label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
-      }
+    if (componentName) {
+      label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
     }
 
     let shouldForwardPropOption = shouldForwardProp;
diff --git a/node_modules/@mui/system/esm/createStyled.js b/node_modules/@mui/system/esm/createStyled.js
index fad28cd..d5aa72d 100644
--- a/node_modules/@mui/system/esm/createStyled.js
+++ b/node_modules/@mui/system/esm/createStyled.js
@@ -95,10 +95,8 @@ export default function createStyled(input = {}) {
     const skipSx = inputSkipSx || false;
     let label;
 
-    if (process.env.NODE_ENV !== 'production') {
-      if (componentName) {
-        label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
-      }
+    if (componentName) {
+      label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
     }
 
     let shouldForwardPropOption = shouldForwardProp;
diff --git a/node_modules/@mui/system/legacy/createStyled.js b/node_modules/@mui/system/legacy/createStyled.js
index 7814a46..4947636 100644
--- a/node_modules/@mui/system/legacy/createStyled.js
+++ b/node_modules/@mui/system/legacy/createStyled.js
@@ -96,10 +96,8 @@ export default function createStyled() {
     var skipSx = inputSkipSx || false;
     var label;
 
-    if (process.env.NODE_ENV !== 'production') {
-      if (componentName) {
-        label = "".concat(componentName, "-").concat(lowercaseFirstLetter(componentSlot || 'Root'));
-      }
+    if (componentName) {
+      label = "".concat(componentName, "-").concat(lowercaseFirstLetter(componentSlot || 'Root'));
     }
 
     var shouldForwardPropOption = shouldForwardProp;
