diff --git a/node_modules/@mui/system/createStyled.js b/node_modules/@mui/system/createStyled.js
index 6ae7c7c..e717e86 100644
--- a/node_modules/@mui/system/createStyled.js
+++ b/node_modules/@mui/system/createStyled.js
@@ -112,10 +112,8 @@ function createStyled(input = {}) {
     const skipSx = inputSkipSx || false;
     let label;
 
-    if (process.env.NODE_ENV !== 'production') {
-      if (componentName) {
-        label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
-      }
+    if (componentName) {
+      label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
     }
 
     let shouldForwardPropOption = shouldForwardProp;
