diff --git a/node_modules/@mui/system/createStyled.js b/node_modules/@mui/system/createStyled.js
index 1c1ac71..35f365f 100644
--- a/node_modules/@mui/system/createStyled.js
+++ b/node_modules/@mui/system/createStyled.js
@@ -112,10 +112,8 @@ function createStyled(input = {}) {
     const skipSx = inputSkipSx || false;
     let label;
 
-    if (process.env.NODE_ENV !== 'production') {
-      if (componentName) {
-        label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
-      }
+    if (componentName) {
+      label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
     }
 
     let shouldForwardPropOption = shouldForwardProp;
diff --git a/node_modules/@mui/system/esm/createStyled.js b/node_modules/@mui/system/esm/createStyled.js
index 68011a6..932d58e 100644
--- a/node_modules/@mui/system/esm/createStyled.js
+++ b/node_modules/@mui/system/esm/createStyled.js
@@ -93,10 +93,8 @@ export default function createStyled(input = {}) {
     const skipSx = inputSkipSx || false;
     let label;
 
-    if (process.env.NODE_ENV !== 'production') {
-      if (componentName) {
-        label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
-      }
+    if (componentName) {
+      label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
     }
 
     let shouldForwardPropOption = shouldForwardProp;
diff --git a/node_modules/@mui/system/modern/createStyled.js b/node_modules/@mui/system/modern/createStyled.js
index 673f7ae..717f140 100644
--- a/node_modules/@mui/system/modern/createStyled.js
+++ b/node_modules/@mui/system/modern/createStyled.js
@@ -91,10 +91,8 @@ export default function createStyled(input = {}) {
     const skipSx = inputSkipSx || false;
     let label;
 
-    if (process.env.NODE_ENV !== 'production') {
-      if (componentName) {
-        label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
-      }
+    if (componentName) {
+      label = `${componentName}-${lowercaseFirstLetter(componentSlot || 'Root')}`;
     }
 
     let shouldForwardPropOption = shouldForwardProp;
